linkerd/admin/src/main/scala/io/buoyant/linkerd/admin/AdminHandler.scala

```diff
@@ -29,7 +29,7 @@ object AdminHandler extends HtmlView {
             <li><a href="/metrics">metrics</a></li>
             <li><a href="/admin/logging">logging</a></li>
             <li><a href="https://linkerd.io/help/">help</a></li>
-            <!-- <li><a href="/dashboard">Beta</a></li> -->
+            <li><a href="/dashboard">Beta dashboard</a></li>
```

Comment:
... or is this served from the old dashboard.  regardless, i think this link needs to change...